pkg/driver/controller.go

```diff
@@ -924,7 +924,7 @@ func newCreateVolumeResponse(disk *cloud.Disk, ctx map[string]string) *csi.Creat
 		}
```

Comment:
Question: Should we also have to change our examples documentation to the well known key like [in storage class example](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/ec8718afe87806ed5627c6fec54e54240feb5289/examples/kubernetes/storageclass/manifests/storageclass.yaml#L28)?